scripts/get-job-matrix.py

```diff
@@ -133,29 +133,29 @@ class MakefileTest(TypedDict):
 # versions in the the pulumi-docker-containers repo by updating the file
 # https://github.com/pulumi/pulumi-docker-containers/blob/main/.github/scripts/matrix/versions.py
 
-MINIMUM_SUPPORTED_VERSION_SET = {
-    "name": "minimum",
-    "dotnet": "6",
-    "go": "1.23.x",
-    "nodejs": "18.x",
-    # When updating the minimum Python version here, also update `pyproject.toml`, including the
-    # `mypy` and `ruff` sections.
-    "python": "3.9.x",
-}
-
 ALL_VERSION_SET = {
-    "dotnet": ["6", "8", "9"],
+    "dotnet": ["8", "9"],
     "go": ["1.23.x", "1.24.x"],
     "nodejs": ["18.x", "20.x", "22.x", "23.x"],
+    # When updating the minimum Python version here, also update `pyproject.toml`, including the
+    # `mypy` and `ruff` sections.
     "python": ["3.9.x", "3.10.x", "3.11.x", "3.12.x", "3.13.x"],
 }
 
+MINIMUM_SUPPORTED_VERSION_SET = {
+    "name": "minimum",
+    "dotnet": ALL_VERSION_SET["dotnet"][0],
+    "go": ALL_VERSION_SET["go"][0],
+    "nodejs": ALL_VERSION_SET["nodejs"][0],
+    "python":   ALL_VERSION_SET["python"][0],
```

Comment:
Consider populating dynamically to avoid possibly mismatching the keys.

```suggestion
    **{lang: versions[0] for lang, versions in ALL_VERSION_SET.items()}
```

Reply:
Even better